dsgrid/dimension/standard.py

```diff
@@ -24,10 +24,16 @@
 # ---------------------------
 # GEOGRAPHIC DIMENSIONS
 # ---------------------------
+class Geography(GeographyDimensionBaseModel):
+    """Generic geography with optional time_zone"""
```

Comment:
@daniel-thom, a simple geography class (`GeographyDimensionBaseModel`) was nominally already available. This just makes it more obvious (and with a simpler name).